osu.Game/Skinning/SkinnableSamples.cs

```diff
@@ -266,10 +264,18 @@ private void cancelPendingStart()
 
         public void RemoveAllAdjustments(AdjustableProperty type) => samplesContainer.RemoveAllAdjustments(type);
 
+        /// <summary>
+        /// Whether sample playback was requested via a call to <see cref="Play"/>.
+        ///
+        /// Importantly, this will ignore the current gameplay paused state (samples may be queued to be played after unpause).
+        /// For the imminent playback state, use <see cref="IsActivelyPlaying"/>.
+        /// </summary>
+        public bool IsPlaying { get; private set; }
+
         /// <summary>
         /// Whether any samples are currently playing.
         /// </summary>
-        public bool IsPlaying
+        public bool IsActivelyPlaying
```

Comment:
Don't like these naming changes. `RequestedPlaying` + `IsPlaying` is less confusing to me than `IsPlaying` + `IsActivelyPlaying`.